components/lib/crash/src/main/res/values-my/strings.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <!-- Title of the crash reporter dialog. %1$s will be replaced with the name of the app (e.g. Firefox Focus). -->
+    <string name="mozac_lib_crash_dialog_title">ဝမ်းနည်းပါတယ်။ %1$s တွင် ပြဿနာတစ်ခု ပေါ်ခဲ့သဖြင့် ရပ်ဆိုင်းသွားသည်။</string>
+
+    <!-- Label of the checkbox for sending crash reports in the crash reporter dialog. %1$s will be replaced with the name of the organization (e.g. Mozilla). -->
+    <string name="mozac_lib_crash_dialog_checkbox">%1$s သို့ ပျက်စီးမှုအစီရင်ခံစာပို့ပါ</string>
+
+    <!-- Label of the button closing the crash reporter dialog. -->
+    <string name="mozac_lib_crash_dialog_button_close">ပိတ်ပါ</string>
+
+    <!-- Label of the button closing the crash reporter dialog and restarting the app. -->
+    <string name="mozac_lib_crash_dialog_button_restart">ပြန်စပါ</string>
```

Comment:
Lint is complaining about the missing parameter here. I am not sure if we can relax the rule here since I think the opposite can crash an app - I need to test and verify that.

```
[task 2020-03-17T00:14:24.169Z]   /builds/worker/checkouts/src/components/lib/crash/src/main/res/values/strings.xml:16: Error: Inconsistent number of arguments in formatting string mozac_lib_crash_dialog_button_restart; found both 0 and 1 [StringFormatCount]
[task 2020-03-17T00:14:24.169Z]       <string name="mozac_lib_crash_dialog_button_restart">Restart %1$s</string>
[task 2020-03-17T00:14:24.169Z]       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
[task 2020-03-17T00:14:24.169Z]       /builds/worker/checkouts/src/components/lib/crash/src/main/res/values-my/strings.xml:14: Conflicting number of arguments here
```

Reply:
Where do I see this? I clicked on all the failure, the only log I found is [this](https://firefox-ci-tc.services.mozilla.com/tasks/L7SULiinRLuw0OKuXTvBcQ/runs/0/logs/https%3A%2F%2Ffirefox-ci-tc.services.mozilla.com%2Fapi%2Fqueue%2Fv1%2Ftask%2FL7SULiinRLuw0OKuXTvBcQ%2Fruns%2F0%2Fartifacts%2Fpublic%2Flogs%2Flive.log), and it looks like an unrelated failure?

Reply:
On the specific issue: I can reject that translation in Pontoon to see if it unlocks us, but we should definitely look into making the rules less strict. It sounds surprising that the app would crash without a placeable, only the substitution should fail. But, given it's Android, I'm not sure I should be surprised anymore 🤷‍♂

Reply:
> Where do I see this?

Oh, figured it out: build-lib-crash -> firefoxci-taskcluster -> full log.

In the meantime, I've unapproved the string in Pontoon, but automation kicks in only once a day :-\